cnmodel/data/ionchannels.py

```diff
@@ -0,0 +1,332 @@
+
+[1] Uses channels from Rothman and Manis, 2003
+    Conductances are for Mouse bushy cells
+    Xie and Manis, 2013
+    Age "adult", Temperature=34C
+    Units are nS.
+
+""")
+
+
+add_table_data('POK_channels', row_key='field', col_key='cell_type', 
+               species='rat', data=u"""
+
+This table describes the ion channel densities and voltage shifts for rat DCN pyramidal cells,
+from Kanold and Manis, 2001
+
+The table includes 2 additiona variants
+
+-----------------------------------------------------------------------------------------------------------------------------------
+                  pyramidal   
+                           
+soma_napyr_gbar   350.0  [1]      
+soma_nap_gbar     0.
+soma_kdpyr_gbar   80.0   [1]
+soma_kcnq_gbar    0.
+soma_kif_gbar     150.0  [1]
+soma_kis_gbar     40.0   [1]
+soma_ihpyr_gbar   2.8    [1]     
+soma_leak_gbar    2.8    [1]
+soma_leak_erev    -62.0  [1]
+soma_e_na         50.    [1]
+soma_e_k          -81.5  [1]
+soma_e_h          -43.0  [1]
+soma_natype       napyr
+soma_Cap          12     [1]
+
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+[1] Kanold and Manis, 1999, 2001, 2005
+    Age P11-14, Temperature=22C
+    Units are nS.
+
+""")
+
+add_table_data('CW_channels', row_key='field', col_key='cell_type', 
+               species='mouse', data=u"""
+
+This table describes the ion channel densities and voltage shifts
+for a mouse carthweel cell model.
+Ad-hoc model, based on a Purkinje cell model (ref [1]).
+
+
+-----------------------------------------------------------------------------------------------------------------------------------
+                   cartwheel   
+                           
+soma_narsg_gbar    500.0  [1]     
+soma_bkpkj_gbar    2.0
+soma_kpkj_gbar     100.   [1]
+soma_kpkj2_gbar    50.
+soma_kpkjslow_gbar 150    [1]
+soma_kpksk_gbar    25.0   [1]
+soma_lkpkj_gbar    5.0    [1]     
+soma_hpkj_gbar     5.0    [1]
+soma_e_na          50.    [1]
+soma_e_k           -80.0  [1]
+soma_hpkj_eh       -43.0  [1]
+soma_lkpkj_e       -65.0  [1]
+soma_e_ca          50.
+soma_na_type       narsg
+soma_pcabar        0.00015 [1]
+soma_Dia           18  
+
+-----------------------------------------------------------------------------------------------------------------------------------
+
+[1] Channels from Khaliq, Gouwens and Raman, J. Neurosci. 2003
+    Conductance levels modified. 
+
+""")
+
+add_table_data('TV_channels', row_key='field', col_key='cell_type', 
+               species='mouse', data=u"""
+
+This table describes the ion channel densities and voltage shifts
+for a mouse tuberculoventral cell model.
+Ad-hoc model, based on the t-stellate cell model, but adjusted
+to match the data from Kuo and Trussell.
+
+            self.set_soma_size_from_Cm(35.0)
```

Comment:
I guess you meant to remove this.

Reply:
Yes, and I have removed this. It will be corrected in the next PR.